src/robots.cpp

```diff
@@ -55,7 +55,7 @@ namespace Rep
 
     Robots::Robots(const std::string& content): agents_(), sitemaps_(), default_(agents_["*"])
     {
-        std::string agent_name("");
+        std::string agent_name("*");
```

Comment:
It would be nice to distinguish between normal declared `*` agents and top-level undeclared agents for the purposes of reporting site issues.